packages/client-assertion-validation/test/validation.test.ts

```diff
@@ -570,48 +547,60 @@ describe("validation test", () => {
     });
   });
 
-  describe("verifyClientAssertionSignature", () => {
-    it("success client assertion signature", () => {
+  describe("verifyClientAssertionSignature", async () => {
+    it("success client assertion signature", async () => {
       const threeHourLater = new Date();
       threeHourLater.setHours(threeHourLater.getHours() + 3);
 
-      const keySet = crypto.generateKeyPairSync("rsa", {
-        modulusLength: 2048,
-      });
-
-      const jws = getMockClientAssertion({
+      const { jws, publicKeyEncodedPem } = await getMockClientAssertion({
         customHeader: {},
         standardClaimsOverride: {
           iat: new Date().getTime() / 1000,
           exp: threeHourLater.getTime() / 1000,
         },
         customClaims: {},
-        keySet,
-      });
-      const publicKey = keySet.publicKey
-        .export({
-          type: "pkcs1",
-          format: "pem",
-        })
-        .toString();
+      });
       const mockConsumerKey = {
-        ...getMockConsumerKey(),
-        publicKey,
+        ...getMockKey(),
```

Comment:
this and following occurrences:
it is not required for this test to be a consumerKey, but to avoid confusion if the const is called `mockConsumerKey` we should use the `getMockConsumerKey` helper (otherwise we can just rename the const)